app/controllers/application_controller.rb

```diff
@@ -5,6 +5,21 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "another_super_duper_secret_phrase"
   end
 
-end
\ No newline at end of file
+  get '/' do
+    erb :index
+  end
+
+  helpers do
+    def logged_in?
+      !!session[:user_id]
+    end
+
+    def current_user
+      User.find(session[:user_id])
```

Comment:
@mendelB -- just to verify, that @current_user instance variable will be visible in all the controllers that inherit from ApplicationController and (therefore) all views, correct?